sys/include/net/gnrc/lowpan/frag.h

```diff
@@ -1,76 +1,77 @@
 /*
- * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
- * Copyright (C) 2015 Hamburg University of Applied Sciences
+ * Copyright (C) 2015, 2017     Hamburg University of Applied Sciences
```

Comment:
cut it short to _HAW Hamburg_ would be (more) consistent with recent copyrights.